src/parquet/file/metadata.h

```diff
@@ -91,6 +91,32 @@ class PARQUET_EXPORT RowGroupMetaData {
 
 class FileMetaDataBuilder;
 
+struct PARQUET_EXPORT FileVersion {
```

Comment:
Should we move this to a separate file `util/file-version.h` / `util/file-version.cc`